vagrant/post-deploy.d/00-run-docker.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+set -eu
+
+PAAS_DIR=/tmp/paas-bootstrap
+
+sudo apt update && sudo apt install docker-compose -y
```

Comment:
Have we looked into using an AMI with docker included so we can avoid any installation?